dockerfiles/redis/Dockerfile

```diff
@@ -0,0 +1,7 @@
+

```

Comment:
Don't add a newline at the start of the file

Reply:
Acknowledged sir